docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -2,17 +2,33 @@ import * as React from 'react';
 import { alpha } from '@mui/material/styles';
 import Link from 'docs/src/modules/components/Link';
 import FEATURE_TOGGLE from 'docs/src/featureToggle';
+import PageContext from 'docs/src/modules/components/PageContext';
+import { convertProductIdToName } from 'docs/src/modules/components/AppSearch';
 
 export default function AppFrameBanner() {
+  const pageContext = React.useContext(PageContext);
+
+  const productName = convertProductIdToName(pageContext) || 'MUI';
+  const message = `Influence ${productName}'s 2024 roadmap! Take a few minutes to answer the latest Developer Survey`;
```

Comment:
```suggestion
  const message = `Influence ${productName}'s 2024 roadmap! Participate in the latest Developer Survey`;
```

Maybe we can cut it short like this? Wdyt @samuelsycamore?